ios/Framework/Core/Base/BaseConnectors/AsyncServerConnector.swift

```diff
@@ -0,0 +1,83 @@
+/**
+* Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+*
+* This library is free software; you can redistribute it and/or modify it under
+* the terms of the GNU Lesser General Public License as published by the Free
+* Software Foundation; either version 2.1 of the License, or (at your option)
+* any later version.
+*
+* This library is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+* FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+* details.
+*/
+
+import Foundation
+
+@objc(AsyncServerConnector)
+open class AsyncServerConnector: ServerConnector {
+
+	fileprivate var _finished: Bool = false
+
+	fileprivate var _executing: Bool = false
+
```

Comment:
I didn't realize this, but we don't use underscore for any variable

Reply:
Right! I'll change it 👍

Reply:
I tried to change the names but there are two properties with the same name in `Operation` class, so, for now, those variables are going to have the underscore.

Reply:
👍